src/main/java/com/github/bartimaeusnek/cropspp/crops/TC/CinderpearlCrop.java

```diff
@@ -34,28 +34,19 @@ public String discoveredBy() {
 
     @Override
     public int growthDuration(ICropTile crop) {
-        int r;
-        if (ConfigValues.debug)
-            r = 1;
-        else if (crop.getSize() == 1)
-            r = 2250;
-        else if (crop.getSize() == 2 && (crop.isBlockBelow("blockBlaze") || !(OreDictionary.doesOreNameExist("blockBlaze"))))
-            r = 1750;
-        else
-            r = 250;
-        return r;
+        if (ConfigValues.debug) return 1;
+        return crop.getSize() == 1 ? 2250 : 1750;
```

Comment:
Is it intended that the check for a blaze block below is missing?

Reply:
Yes, that's handled in [canGrow()](https://github.com/GTNewHorizons/Crops-plus-plus/blob/master/src/main/java/com/github/bartimaeusnek/cropspp/crops/TC/CinderpearlCrop.java#L57)